src/forms/LoginForm.php

```diff
@@ -4,6 +4,7 @@
 
 use Yii;
 use yii\base\Model;
+use app\models\User;
```

Comment:
While this is currently required for the template to work I'd prefer to remove the `User` model and login completely and rather have something in the docs which describes how to install a full-featured module such as `yii2-usuario`.

Reply:
I don't think it is very wise to have a demo app which does not work or throws an error, it should be work out of the box and docs should indicate how to use a different user module.

Reply:
https://github.com/yiisoft/app/issues/40

Reply:
Not a question of the current PR for sure.